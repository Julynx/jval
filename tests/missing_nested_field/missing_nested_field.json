{
    "role": "user",
    "name": "Alice",
    "cars": [
        {
            "type": "car",
            "brand": "Toyota",
            "model": "Corolla",
            "year": 2020,
            "properties": [
                {
                    "city": "New York",
                    "street": "5th Avenue",
                    "valued_at": 1000000
                }
            ]
        }
    ]
}