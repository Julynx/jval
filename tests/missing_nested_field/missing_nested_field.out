Validation error at 'cars[0].properties[0].type': missing value for key 'type'